js/main.js

```diff
@@ -19,19 +19,17 @@ $startButton.addEventListener('click', function (event) {
 });
 
 document.addEventListener('click', function (event) {
-  for (var i = 0; i < $searchIcon.length; i++) {
-    if (event.target === $searchIcon[i]) {
-      $homePage.className = 'home-page hidden';
-      $mainPage.className = 'main-page';
-      $activityPage.className = 'activity-page hidden';
-      $levelPage.className = 'level-page hidden';
-    }
-    if (event.target === $homeIcon[i]) {
-      $homePage.className = 'home-page';
-      $mainPage.className = 'main-page hidden';
-      $activityPage.className = 'activity-page hidden';
-      $levelPage.className = 'level-page hidden';
-    }
+  if (event.target === $searchIcon) {
+    $homePage.className = 'home-page hidden';
+    $mainPage.className = 'main-page';
+    $activityPage.className = 'activity-page hidden';
+    $levelPage.className = 'level-page hidden';
+  }
+  if (event.target === $homeIcon) {
+    $homePage.className = 'home-page';
+    $mainPage.className = 'main-page hidden';
+    $activityPage.className = 'activity-page hidden';
+    $levelPage.className = 'level-page hidden';
```

Comment:
Last change that should be done for the sake of optimization.

Instead of multiple if statements, you should have if and elseif statements.

The problem with having multiple if statements is that even if one condition is met, the computer will have to check every single conditional.  Which doesn't really make sense because `event.target` will always be one dom element.

If you have if else if statements, then once a condition is met, it won't try to check the others as well.